Assets/Python/CityNameManager.py

```diff
@@ -5626,14 +5706,84 @@ def onRevolution(iCiv):
 #Language: Korean
 {
 	"Beijing"		:	"Bukgyeong",
-	"Zhongdu"		:	"Bukgyeong",
+	"Zhongdu"		:	"Bukpyeong",
 	"Chunju"		:	"Chunju",
-	"Guangzhou"		:	"Gwangju",
+	"Guangzhou"		:	"Guangju",
 	"Hanseong"		:	"Hanseong",
-	"Pusan"			:	"Pusan",
+	"Dongrae"		:	"Dongrae",
+	"Busan"			:	"Busan",
 	"Pyongyang"		:	"Pyongyang",
-	"Shenyang"		:	"Seonyang",
+	"Shenyang"		:	"Shimyang",

```

Comment:
That's not how I usually see that transliterated.

Reply:
There are two ways to read the Chinese letter 'Shen'.
The appropriate way in Korea is 'Shim'.
A minor fix.

Reply:
Guangzhou - Guangju is to avoid confusion with Gwangju in the Korean penn.
Bukpyeong is the translation for Beiping, one of the ancient/medieval names of Beijing.